src/tools/perf/uct_tests.cc

```diff
@@ -335,6 +335,13 @@ class uct_perf_test_runner {
         return UCS_OK;
     }
 
+    void wait_for_window(bool send_window)
```

Comment:
I think it is better to place the function near progress_requestor() in line  126.